### PR TITLE
Loomerings

### PR DESCRIPTION
**Pull request by romangraef**
(This is purely a buildscript change). I also want to stress that we **are not** using any essentials components at runtime, we are just using a version of architectury which has been modified by sk1er at compile time.

### CODE REVIEW

**Review by Lulonaut — APPROVED** [review]
looming

File: src/main/java/io/github/moulberry/notenoughupdates/overlays/RancherBootOverlay.java [hunk 1]
```diff
@@ -31,7 +32,7 @@ public static boolean shouldReplace() {
 
 		if (!(Minecraft.getMinecraft().currentScreen instanceof GuiEditSign)) return false;
 
-		TileEntitySign tes = ((GuiEditSign) Minecraft.getMinecraft().currentScreen).tileSign;
+		TileEntitySign tes = ((AccessorGuiEditSign) Minecraft.getMinecraft().currentScreen).getTileSign();
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/overlays/RancherBootOverlay.java:35]: rancher boot overlay update PauseChamp

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/overlays/RancherBootOverlay.java:35]: we have a rancher boot overlay?

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/overlays/RancherBootOverlay.java:35]: also do actual review

Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/overlays/RancherBootOverlay.java:35]: > we have a rancher boot overlay?

we dont

Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/overlays/RancherBootOverlay.java:35]: its just a dead old class

Comment (NopoTheGamer) [reply src/main/java/io/github/moulberry/notenoughupdates/overlays/RancherBootOverlay.java:35]: > also do actual review

i will when I am not in class :bangbang:

**Review by NopoTheGamer — APPROVED** [review]

**Pull request merged**